modules/map/relative_map/relative_map_interface.h

```diff
@@ -47,13 +47,23 @@ class RelativeMapInterface : public apollo::common::ApolloApp {
   virtual void RunOnce() = 0;
 
   /**
-   * @brief Fill the header and publish the prediction message.
+   * @brief Fill the header and publish the relativemap message.
    */
   void Publish(MapMsg *relative_map) {
     apollo::common::adapter::AdapterManager::FillRelativeMapHeader(
         Name(), relative_map);
     apollo::common::adapter::AdapterManager::PublishRelativeMap(*relative_map);
   }
+
+  /**
+   * @brief Fill the header and publish the navigation info message.
+   */
+  void PublishNavigationInfo(NavigationInfo* const navigation_info) {
```

Comment:
Put it into the .cc file inside anonymous namespace if the .cc is the only user.

Reply:
I made it as a member function of class RelativeMap, declared in relative_map.cc. Please review again, thx.